olive/cli/perf_tuning.py

```diff
@@ -107,61 +78,36 @@ def register_subcommand(parser: ArgumentParser):
                 'E.g. --extra_session_config \'{"key1": "value1", "key2": "value2"}\''
             ),
         )
-        pass_group.add_argument(
+        sub_parser.add_argument(
             "--disable_force_evaluate_other_eps",
             action="store_true",
             help=(
                 "Whether force to evaluate all execution providers"
                 " which are different with the associated execution provider."
             ),
         )
-        pass_group.add_argument(
+        sub_parser.add_argument(
             "--enable_profiling",
             action="store_true",
             help="Whether enable profiling for ONNX Runtime inference.",
         )
 
-        # remote options
-        add_remote_options(sub_parser)
+        sub_parser.add_argument(
+            "--predict_with_kv_cache",
+            action="store_true",
+            help="Whether to use key-value cache for ORT session parameter tuning",
+        )
 
+        add_accelerator_options(sub_parser, single_provider=False)
+        add_remote_options(sub_parser)
+        add_logging_options(sub_parser)
         sub_parser.set_defaults(func=PerfTuningCommand)
 
     @staticmethod
     def perf_tuning_template():
```

Comment:
Do you want the template defined in this file rather a separate template file? If so we probably can remove this method and the template file as well.

Reply:
In fact, we're not using this template defined in a separate file in the CLI anymore. Removed